src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -26,8 +26,10 @@ internal static class DependentProjectsFinder
         /// </summary>
         private struct DefinitionProject
         {
+#pragma warning disable IDE0052 // Remove unread private members
             private readonly ProjectId _sourceProjectId;
             private readonly string _assemblyName;
+#pragma warning restore IDE0052 // Remove unread private members
```

Comment:
why?

Reply:
`DefinitionProject` is used as a key in the dictionary below, I'll add a comment.

Reply:
`DefinitionProject` does not implement `IEquatable<DefinitionProject>`. If you want to suppress IDE0052 instead of fixing it, the type needs to be renamed to `DefinitionProjectWillMakeYourDictionarySlow` at the same time. 😄